nx.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
```

Comment:
What is this?

Reply:
This is the JSON schema for the `nx.json` file - in particular, it's useful because this allows editors to know what valid keys and values are when editing this file.

Reply:
Gotcha, so this is NX specifying what keys are allowed in their config files?

Reply:
Basically, yeah! Or more specifically, us telling the editor where to look for that information, which winds up being provided by Nx.